examples/pom.xml

```diff
@@ -4,7 +4,7 @@
   <groupId>com.adobe.platform</groupId>
   <artifactId>ecosystem-examples</artifactId>
   <packaging>jar</packaging>
-  <version>1.0.10</version>
+  <version>1.0.12</version>
```

Comment:
Let's use `1.1.0` for this release as it's non breaking release though greater impact than just a patch version bump.